### PR TITLE
Added backwards compatibility for creating TelemetryClient without using dependency injection.

### PR DESCRIPTION
**Pull request by dnduffy**
Added backwards compatibility for creating TelemetryClient without using dependency injection.

### CODE REVIEW

**Comment by SergeyKanzhelev** [comment]
CC: @abaranch @karting

Looks good for me.

**Comment by AlexBulankou** [comment]
:shipit:

File: test/EmptyApp.FunctionalTests/Startup.cs [hunk 1]
```diff
@@ -25,7 +25,7 @@ public void ConfigureServices(IServiceCollection services)
 
             var builder = new ConfigurationBuilder();
             builder.AddApplicationInsightsSettings(instrumentationKey: "Foo");
-            services.AddApplicationInsightsTelemetry(builder.Build());
+            services.AddApplicationInsightsTelemetry(builder.Build(), false);
```
Comment (kartang) [inline test/EmptyApp.FunctionalTests/Startup.cs]: Can we leave this to true, and create a new functional test that would test if telemetry client is sending out the telemetry right? The telemetry client that we use today in the test is the one that we  get from services. At the least, we can send duplicate telemetry from two telemetry clients, and verify if we get it right.

**Comment by kartang** [comment]
Looks good!

File: test/EmptyApp.FunctionalTests/Startup.cs (hunk 1 above)
Comment (dnduffy) [inline test/EmptyApp.FunctionalTests/Startup.cs]: We'll create a separate pull request for additional functional tests.

**Pull request merged**